ddtrace/contrib/rq/__init__.py

```diff
@@ -116,6 +119,9 @@
     ),
 )
 
+rq_version_str = getattr(rq, "__version__", "0.0.0")
+rq_version = parse_version(rq_version_str)
```

Comment:
```suggestion
def _get_version(rq):
    rq_version_str = getattr(rq, "__version__", "0.0.0")
    rq_version = parse_version(rq_version_str)
```

you could also do something like this instead, since the two places where you are trying to use the version you have access to the `rq` module already.

maybe not idea for the `traced_queue_enqueue_job` (to recompute/parse every time), but would fit this setup better.